code/src/components/PopularList/PopularList.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import './PopularList.css'
+
+export const PopularList = () => {
+  const [list, setList] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+    fetch('https://api.themoviedb.org/3/movie/popular?api_key=bdc909b4c34b17568b8111077d9d4d62&language=en-US&page=1')
```

Comment:
We also have the API key in here hehe, but I think it should be hidden. Until we get it to work you could break out the API key as a variable, to make it easier to hide later.